web/css/newberryFrenchSkin.css

```diff
@@ -102,7 +102,7 @@ body{
     padding: 25px 15px 0px 15px;
     position: relative;
     text-align: center;
-    height: 795px; /* however tall it needs to be to fit all text and buttons on small screen scroll */
```

Comment:
There are variables for color that would be better here.

Reply:
We could, I just did it because there is a "light blue" background that happens from heck knows where, which is actually what let us know this was the wrong size, like so (look towards the bottom)

![image](https://user-images.githubusercontent.com/3287006/176965815-72e4073b-9738-4209-bb62-ef6ff98571c9.png)

Fixing the height actually eliminated the need to control the background color at all, we shouldn't have to worry about this any more.